### ISSUE TITLE
Minishift does not install .remove file for add-ons which support it

### ISSUE THREAD

**Issue by agajdosi** [minishift/minishift#1378]
Using binary from latest master, with deleted .minishift folder.

I run any command, it installs the add-ons automatically, but I am then unable to see any `admin-user.addon.remove` file in `.minishift/addons/admin-user` folder. Same result is when I install add-ons manually:

```
[agajdosi@dhcp-24-156 bin]$ ./minishift addon install --defaults
Default add-ons anyuid, admin-user, xpaas, registry-route installed
[agajdosi@dhcp-24-156 bin]$ ./minishift addon remove admin-user
No admin-user.addon.remove file found for admin-user add-on.
```

**Comment by gbraad** [comment]
I think we do not expect these to be deleted... but good point. I believe we should.

**Comment by agajdosi** [comment]
`I think we do not expect these to be deleted`

@gbraad Yes, I was thinking the same, but then I saw that there is admin-user.addon.remove in `minishift/addons/admin-user` and it is not being installed. And so the addon can't be removed.

**Comment by praveenkumar** [comment]
@agajdosi @gbraad I will take a look into it.

**Comment by praveenkumar** [comment]
@agajdosi I am not sure how you get this issue below is what I did and looks everything as expected.

```
$ rm -fr ~/.minishift/

$ ./minishift addons install --defaults
-- Installing default add-ons ... OK
Default add-ons anyuid, admin-user, xpaas, registry-route installed

$ ls ~/.minishift/addons/admin-user/
admin-user.addon  admin-user.addon.remove

$ ./minishift addon remove admin-user
Running this command requires an existing 'minishift' VM, but no VM is defined.

$ ./minishift start
-- Checking if KVM driver is installed ...
[...]
You are logged in as:
    User:     developer
    Password: <any value>

To login as administrator:
    oc login -u system:admin

$ ./minishift addon remove admin-user
-- Removing addon 'admin-user':.
admin user deleted
```
```

**Comment by agajdosi** [comment]
With latest master the behaviour is correct and the same as @praveenkumar reported. This was probably due to fact that admin-user.addon.remove got merged on same day as 1.5.0 went out but was not in 1.5.0. My mistake.

Works as expected, closing this issue.
